[EXOD-000] Add beets LP

melondrone opened the pull request:
- Adds **The Monolith LP** bond
- Sets the EXOD-DAI bond to `expired`
  - Doesn't appear in bond list
  - Is still claimable
- Refactor calculation of treasury value on bond page to use treasury value from the graph.
  - Needs to get gOHM price from eth subgraph and gOHM balance from fantom subgraph
![Screenshot from 2022-01-11 22-19-36](https://user-images.githubusercontent.com/86249394/148965409-0e12bf36-f49b-4efa-a21c-c6cc4fd81798.png)

HOVOH reviewed (commented):
Can you add MAI tracking on the treasury graph as well? https://github.com/ExodiaFinance/exodia-subgraph/blob/03ea3503dbec77dbb3d2cab5d696442d2770fc55/schema.graphql#L120.
I also added a gOHM market value which computes balance value + lp value

melondrone commented on src/slices/AppSlice.ts:
```diff
@@ -114,14 +132,16 @@ export const loadGraphData = createAsyncThunk("app/loadGraphData", async () => {
     circSupply,
     totalSupply,
-    treasuryMarketValue,
+    treasuryMarketValue: treasuryMarketValue + treasuryGOhmBalance * gOhmPrice,
```
Should the `treasuryMarketValue` just contain the entire market value hence no need for computations in the UI?

HOVOH replied:
Yes, it should

melondrone commented:
@HOVOH 

I've updated the charts as follows with a few lil improvements chucked in: 
- Treasury Chart and Breakdown (pie) includes:
  - EXOD (wsEXOD and EXOD from the monolith)
  - MAI (total mai)
  - DAI (total dai)
  - gOHM (total gOHM)
  - wFTM (total wFTM)
- RVF chart includes: 
  - DAI RFV
  - MAI RFV
- Added 5 day average lines to debt charts (referenced in title, thought it'd be good to have the line too)
- Added growth of supply tooltip to chart (referenced in title, should be in the tooltip too)
![Screenshot from 2022-01-12 13-53-12](https://user-images.githubusercontent.com/86249394/149076785-ebe6df76-328b-4ddf-af88-05e7cf4b8fcf.png)
![Screenshot from 2022-01-12 13-53-16](https://user-images.githubusercontent.com/86249394/149076794-d848a8cb-1a7c-4bf5-b691-d4035538a276.png)
![Screenshot from 2022-01-12 14-25-17](https://user-images.githubusercontent.com/86249394/149076797-67ddbb0e-cae3-48d5-b1b3-6a30537370b5.png)
